cloud/members/members-list.md

```diff
@@ -69,4 +58,4 @@ create one.
 </procedure>
 
 
-[cloud-login]: https://console.cloud.timescale.com/
+[cloud-login]: https://cloud.timescale.com
```

Comment:
I think for cloud login we want https://console.cloud.timescale.com/ instead of https://cloud.timescale.com because the latter takes you to the website landing page.